webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric.js

```diff
@@ -390,6 +397,27 @@ export class SakaiRubric extends RubricsElement {
       this.cloneRubric(e);
     }
   }
+
+  // SAK-47640 - Get the maximum and minimum possible grade by multiplying the max or min rating by the criterion weight.
+  getMaxPoints(criterions) {
+    return this.getPoints(criterions, Math.max);
+  }
+
+  getMinPoints(criterions) {
+    return this.getPoints(criterions, Math.min);
+  }
+
+  getPoints(criterions, minOrMax) {
+    let totalPoints = 0;
```

Comment:
I know I really annoy people asking this, but can you please leave a blank line before the method body? The rest of the rubrics code does this and I think it makes the code more readable.